integration-test/1491-generic-network-fixes.py

```diff
@@ -0,0 +1,4231 @@
+            # https://www.openstreetmap.org/way/494863068
+            dsl.way(494863068, dsl.tile_diagonal(z, x, y), {
+                'highway': u'trunk',
+                'maxspeed': u'90',
+                'ref': u'А6',
+                'source': u'openstreetmap.org',
+                'surface': u'asphalt',
+            }),
+            dsl.relation(1, {
+                'distance': u'307.0 km',
+                'name:lt': u'Ryga — Daugpilis — Kraslava — Baltarusijos siena',
+                'network': u'lv:national',
+                'ref': u'A6',
+                'route': u'road',
+                'source': u'openstreetmap.org',
+                'type': u'route',
+                'wikidata': u'Q786297',
+                'wikipedia': u'lv:Autoceļš A6 (Latvija)',
+            }, ways=[494863068]),
+        )
+
+        self.assert_has_feature(
+            z, x, y, 'roads', {
+                'id': 494863068,
+                'network': u'LV:national',
+            })
+
+    def test_lvregional(self):
+        import dsl
+
+        z, x, y = (16, 37781, 20376)
+
+        self.generate_fixtures(
+            dsl.is_in('LV', z, x, y),
+            # https://www.openstreetmap.org/way/15717770
+            dsl.way(15717770, dsl.tile_diagonal(z, x, y), {
+                'bicycle': u'yes',
+                'highway': u'primary',
+                'name': u'Daugavpils iela',
+                'ref': u'P55',
+                'source': u'openstreetmap.org',
+            }),
+            dsl.relation(1, {
+                'distance': u'58.0 km',
+                'name': u'Rēzekne — Dagda',
+                'network': u'lv:regional',
+                'ref': u'P55',
+                'route': u'road',
+                'source': u'openstreetmap.org',
+                'type': u'route',
+                'wikidata': u'Q13979190',
+                'wikipedia': u'lv:Autoceļš P55',
+            }, ways=[15717770]),
+        )
+
+        self.assert_has_feature(
+            z, x, y, 'roads', {
+                'id': 15717770,
+                'network': u'LV:regional',
+            })
+
+    def test_mklocal(self):
+        import dsl
+
+        z, x, y = (16, 36653, 24587)
+
+        self.generate_fixtures(
+            dsl.is_in('MK', z, x, y),
+            # https://www.openstreetmap.org/way/219834131
+            dsl.way(219834131, dsl.tile_diagonal(z, x, y), {
+                'highway': u'tertiary',
+                'name': u'Велушина - Острец',
+                'ref': u'04-01',
+                'source': u'openstreetmap.org',
+            }),
+            dsl.relation(1, {
+                'network': u'mk:local',
+                'ref': u'04-01',
+                'route': u'road',
+                'source': u'openstreetmap.org',
+                'type': u'route',
+            }, ways=[219834131]),
+        )
+
+        self.assert_has_feature(
+            z, x, y, 'roads', {
+                'id': 219834131,
+                'network': u'MK:local',
```

Comment:
That's a crazy ref of `04-01`, what should the **shield_text** be on export? Does the 04 signify which "local" area it's in? Should it be modified on export?

Reply:
Yup, that appears to be the only `NN-NN` ref in FYROM... So I'm not sure if it's a mistake or not. I can't find any references to it elsewhere, but since it's only a local road it might not get signed.

I've swapped it out for another example in 150f72a, since all I wanted to test was the transform of `mk:something` to `MK:something`.